### DISCUSSION TITLE
FilamentPHP v3 Model Events

### DISCUSSION THREAD

**Discussion by SketchNI** [filamentphp/filament#8207] [Help]
Hi,

I'm sort of new to using Filament. I have a form that has an `accepted` and `rejected` toggle.

After reading the Form docs, I don't see any way to have the form dispatch an event when the model is updated so that I can dispatch a Notification which sends a `mail` and `database` notification.

Is there a way to do this that I'm just missing? (Bonus points if I can do this through an Observer)

**Comment by danharrin** [comment]
You can do this through an observer like normal (`saved()` event), no changes to Filament needed. Eloquent does it automatically.
